docs/analytics/jenkins_jobs.md

```diff
@@ -271,3 +275,40 @@ analytics-configuration/automation/run-automated-task.sh StudentEngagementCsvFil
   --interval-type weekly \
   --interval "$FROM_DATE-$TO_DATE"
 ```
+
+## Problem Response Reports
+
+Generates learners' problem responses reports; runs daily.
+
+Run this task only if your client requires the Problem Response Reports.
+
+If Course Block data is available, then it is merged into the generated reports.
+
+Run on periodic build schedule, e.g. ` H 20 * * * `.
+
+Note: HKS epodX wanted theirs to run hourly, so for them we use: ` H * * * * `.
+And we add this to [analytics-override.cfg](resources/analytics-override.cfg), to allow hourly partitions:
+
+    [problem-response]
+    partition_format = %Y-%m-%dT%H
+
+Note that it's just the Problem Response data that is re-generated hourly; the Course Blocks and Course List data is
+still configured to use the default daily partition.
+
+Shell build step:
+
+```
+. /home/jenkins/jenkins_env
+export CLUSTER_NAME="ProblemResponseReportWorkflow Cluster"
+cd $HOME
+
+TOMORROW=`date --date="tomorrow" +%Y-%m-%d`
+
+analytics-configuration/automation/run-automated-task.sh ProblemResponseReportWorkflow \
+    --local-scheduler \
+    --marker $HADOOP_S3_BUCKET/intermediate/problem_response/marker`date +%s` \
+    --interval-end "$TOMORROW" \
+    --hive-overwrite \
+    --n-reduce-tasks $NUM_REDUCE_TASKS
```

Comment:
Does the interval start time default to the current day? It doesn't need to be set to something else explicitly for this to work, right?